src/tools/runk/src/main.rs

```diff
@@ -25,7 +25,7 @@ enum SubCommand {
     #[clap(flatten)]
     Standard(StandardCmd),
     #[clap(flatten)]
-    Common(CommonCmd),
+    Common(Box<CommonCmd>),
```

Comment:
@mayank-mrinal-sony 
Could you add a comment to explain the reason why `Box` is needed?
The comment will be helpful for developers to understand the necessity.